### ISSUE TITLE
display image in MACachedImageView without using cache

### ISSUE THREAD

**Issue by JanC** [swissmanu/MACachedImageView#2]
Hi,
I use MACachedImageView to download images from the net but in some cases, I actually do have the image in my application. In that case, I don't want to use any caching and just display the image in the MACachedImageView. How can I do that?

I tried using the placeholderImage property:

```
cell.cachedImageView.placeholderImage = image;
```

but this will not work. At least not all the time. I think it this happens when the MACachedImageView was already used to display a cached image (the cell is reused and so is the `cell.cachedImageView`

cheers

**Comment by JanC** [comment]
ok, basically, I need this method to be public:

```
-(void)displayImage:(UIImage*)image withContentMode:(UIViewContentMode)contentMode;
```

or, the placeHolderImage setter should display the image all the time:

```
-(void)setPlaceholderImage:(UIImage *)placeholderImage {
    _placeholderImage = placeholderImage;

     // do not do this check
    //if(!_displayingImage) {
        // Ensure the new placeholder image is properly displayed.
        [self hideImage];
    //}
}
```

**Comment by swissmanu** [comment]
thanks for your input @JanC.

i just released version `1.1.0` of `MACachedImageView` which provides a `displayImage:(UIImage*)image` method now.

cheers,
manu

**Comment by JanC** [comment]
great, I'll test it
